spec/models/route_stop_pattern_spec.rb

```diff
@@ -0,0 +1,371 @@
+# == Schema Information
+#
+# Table name: current_route_stop_patterns
+#
+#  id                                 :integer          not null, primary key
+#  onestop_id                         :string
+#  geometry                           :geography({:srid geometry, 4326
+#  tags                               :hstore
+#  stop_pattern                       :string           default([]), is an Array
+#  version                            :integer
+#  created_or_updated_in_changeset_id :integer
+#  is_generated                       :boolean          default(FALSE)
+#  is_modified                        :boolean          default(FALSE)
+#  is_only_stop_points                :boolean          default(FALSE)
+#  trips                              :string           default([]), is an Array
+#  identifiers                        :string           default([]), is an Array
+#  created_at                         :datetime         not null
+#  updated_at                         :datetime         not null
+#  route_id                           :integer
+#
+# Indexes
+#
+#  index_current_route_stop_patterns_on_identifiers  (identifiers)
+#  index_current_route_stop_patterns_on_route_id     (route_id)
+#
+
+describe RouteStopPattern do
+  before(:each) do
+    create(:stop,
+      onestop_id: "s-9q8yw8y448-bayshorecaltrainstation",
+      geometry: point = Stop::GEOFACTORY.point(-122.401811, 37.706675).to_s
```

Comment:
Use default factory if possible, refer by let: name. Also take out assignment to point.